examples/with-auth/README.md

```diff
@@ -0,0 +1,21 @@
+# Integrating with `@urql/exchange-auth`'s authExchange
```

Comment:
Just a small note, I'd call this example `with-refresh-auth` or something of the likes to make sure the name isn't a catch all (ie if someone just adds a token without refresh)

Reply:
sounds good